rules/windows/credential_access_mimikatz_memssp_default_logs.toml

```diff
@@ -11,6 +11,51 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Mimikatz Memssp Log File Detected"
+note = """## Triage and analysis.
+
+### Investigating Mimikatz Memssp Log File Detected
+
+[Mimikatz](https://github.com/gentilkiwi/mimikatz) is an open-source tool used to collect, decrypt, and/or use cached
+credentials. This tool is commonly abused by adversaries during the post-compromise stage where adversaries have gained
+an initial foothold onto an endpoint and are looking to elevate privileges and seek out additional authentication objects
+such as tokens/hashes/credentials that can then be used to laterally move and pivot across a network.
+
+This rule looks for the creation of a file named `mimilsa.log`, which is generated when using the Mimikatz misc::memssp
+module, which injects a malicious Windows SSP to collect locally authenticated credentials, which includes the computer
+account password, running service credentials, and any accounts that logon.
+
+#### Possible investigation steps:
+
+- Investigate script execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Scope potentially compromised accounts. Analysts can do this by searching for login events (e.g., 4624) to the target
+host.
+- Retrieve and inspect the log file contents.
+- By default, the log file is created in the same location as the DLL file.
```

Comment:
What location is this usually?

Reply:
System32, but I think the attacker can use other locations as well

Reply:
So I want the analyst to analize the same directory in which the log file was created, that may contain the DLL used